subprojects/language-native/src/integTest/groovy/org/gradle/language/cpp/CppToolChainChangesIntegrationTest.groovy

```diff
@@ -85,30 +131,13 @@ class CppToolChainChangesIntegrationTest extends AbstractIntegrationSpec {
                 }
             }
         """
-
-        when:
-        run ':app:compileDebugCpp'
-
-        then:
-        executedAndNotSkipped ':app:compileDebugCpp'
-
-        when:
-        run ':app:compileDebugCpp', '-PuseAlternativeToolChain=true', "--info"
-
-        then:
-        executedAndNotSkipped ':app:compileDebugCpp'
-        output =~ /Value of input property 'compilerVersion\.(version|type)' has changed for task ':app:compileDebugCpp'/
-
-        where:
-        toolChains << toolChainPairs
-        toolChainBefore = toolChains[0]
-        toolChainAfter = toolChains[1]
     }
 
-    private static List<List<AvailableToolChains.InstalledToolChain>> getToolChainPairs() {
+    private static List<List<InstalledToolChain>> getToolChainPairs() {
         def availableToolChains = AvailableToolChains.toolChains.findAll {
             it.available && worksWithCppPlugin(it) && !(it instanceof AvailableToolChains.InstalledSwiftc)
         }
+        Assume.assumeTrue("At least two toolchains available", availableToolChains.size() > 1)
```

Comment:
I think this line is now unnecessary due to the change a couple of lines below.  We can keep either one.

Reply:
I removed my new one.